api/v1alpha1/etcd.go

```diff
@@ -233,6 +233,22 @@ type EtcdConfig struct {
 	// ClientService defines the parameters of the client service that a user can specify
 	// +optional
 	ClientService *ClientService `json:"clientService,omitempty"`
+	// ExistingCluster specifies the configuration for an existing etcd cluster that the etcd-druid managed cluster will join.
```

Comment:
Is it possible to make this field immutable? Once the `Etcd` spec has been created, `Spec.Etcd.ExistingCluster` can no longer be added or modified.

Also, once the new cluster has joined the existing cluster, will this field still be retained in the Etcd spec? If yes, then what's the behavior of etcd-druid after the user manually removes the old cluster members (assuming they wanted druid to "adopt" their existing cluster)? If `Spec.Etcd.ExistingCluster` still exists after removal of the old members, will druid still attempt to create the cluster with the old members also, which will lead to a quorum loss since only 3 out of total 6 members will now exist?

I think we need some discussion/clarity on this behavior.